docker-compose.yml

```diff
@@ -1,5 +1,16 @@
 version: "3"
 services:
+  fastapi_app:
+    container_name: fastapi_app
+    image: emrecetinkayadev/fastapi-demo-project:1.0.0
+    build: .
+    ports:
+      - "8000:8000"
+    environment:
+      - DB_URL=postgresql://postgres:password123@postgres:5432/fastapi
```

Comment:
Try using pydantic to store secrets, connection strings, etc.
https://progressstory.com/tech/configuration-management-python-pydantic/
(See chapter 'Environment variable combined with Class-based configuration')